lib/internal/net.js

```diff
@@ -57,14 +54,12 @@ function makeSyncWrite(fd) {
 
     this._handle.bytesWritten += chunk.length;
 
-    const ctx = {};
-    writeBuffer(fd, chunk, 0, chunk.length, null, undefined, ctx);
-    if (ctx.errno !== undefined) {
-      const ex = new UVException(ctx);
-      ex.errno = ctx.errno;
-      return cb(ex);
+    try {
+      writeBuffer(fd, chunk, 0, chunk.length, null);
+      cb();
```

Comment:
This would now call `cb()` twice if the first invocation of it throws – is that intentional?